### PR TITLE
[web] unaccent search

### PR DESCRIPTION
**Pull request by mgh-odoo**
- while searching from the different accent characters result is not found.
  this commit will fix that issue

Task: https://www.odoo.com/web#id=1878513&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720
Pad: https://pad.odoo.com/p/r.f86081c98054d24d972902f174e14921

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/core/utils.js [hunk 1]
```diff
@@ -12,6 +12,197 @@ var translation = require('web.translation');
...
+'\u2C73': 'w',
+'\u0078': 'x','\u24E7': 'x','\uFF58': 'x','\u1E8B': 'x','\u1E8D': 'x',
+
+'\u0079': 'y','\u24E8': 'y','\uFF59': 'y','\u1EF3': 'y','\u00FD': 'y','\u0177': 'y','\u1EF9': 'y','\u0233': 'y','\u1E8F': 'y','\u00FF': 'y',
+'\u1EF7': 'y','\u1E99': 'y','\u1EF5': 'y','\u01B4': 'y','\u024F': 'y','\u1EFF': 'y',
+
+'\u007A': 'z','\u24E9': 'z','\uFF5A': 'z','\u017A': 'z','\u1E91': 'z','\u017C': 'z','\u017E': 'z','\u1E93': 'z','\u1E95': 'z','\u01B6': 'z',
+'\u0225': 'z','\u0240': 'z','\u2C6C': 'z','\uA763': 'z',
+};
+
```
Comment (switch87) [inline addons/web/static/src/js/core/utils.js]: I found your answer on stack overflow, strange that you are the first person in 5 years that removes the iteration over a map to make a new map. Nicely done. :+1:

**Comment by switch87** [comment]
@qle-odoo can you ask your good friend robodoo to merge this?

**Comment by qle-odoo** [comment]
@robodoo r+

File: addons/web/static/src/js/core/utils.js [hunk 2]
```diff
@@ -413,6 +604,19 @@ var utils = {
...
         }
     },
+    /**
+     * Replace diacritics character with ASCII character
+     *
+     * @param {string} str diacritics string
+     * @param {boolean} casesensetive
+     * @returns {string} ASCII string
+     */
+    unaccent: function (str, casesensetive=false) {
```
Comment (aab-odoo) [inline addons/web/static/src/js/core/utils.js]: @mgh-odoo @switch87 Default function parameter is an ES6 feature, we can't use it (because of ie 11)

Comment (mgh-odoo) [reply addons/web/static/src/js/core/utils.js]: @aab-odoo 
for the quick merge, I've created PR based on your suggestion. could you please review and merge it?
PR: https://github.com/odoo/odoo/pull/30751

@switch87 @msh-odoo 
Thanks!

Comment (aab-odoo) [reply addons/web/static/src/js/core/utils.js]: :+1:

Comment (msh-odoo) [reply addons/web/static/src/js/core/utils.js]: Thanks @aab-odoo for drawing attention

**Pull request closed without merging**